compiler/qsc_qasm3/src/semantic/tests/decls/bit.rs

```diff
@@ -11,12 +11,12 @@ fn with_no_init_expr_has_generated_lit_expr() {
         "bit a;",
         &expect![[r#"
             ClassicalDeclarationStmt [0-6]:
-                symbol_id: 6
+                symbol_id: 8
                 ty_span: [0-3]
                 init_expr: Expr [0-0]:
                     ty: Bit(true)
-                    kind: Lit: Int(0)
-            [6] Symbol [4-5]:
+                    kind: Lit: Bit(0)
+            [8] Symbol [4-5]:
```

Comment:
Test below should be able to be run now (removing ignore):
#[ignore = "Unimplemented"]
fn array_with_no_init_expr_has_generated_lit_expr() {